app/templates/components/doi-related-identifier.hbs

```diff
@@ -0,0 +1,122 @@
+{{#if (gt index 0)}}
+  <div class="power-select-label">
+    <label for="relatedIdentifier" class="subtitle">Related Identifier</label> 
+  </div> 
+<div class="input-group">
+
+  <input type="text"
+    class="form-control related-identifier-field no-error {{if (not fragment.relatedIdentifier) 'no-success'}}" value={{fragment.relatedIdentifier}} oninput={{action "updateRelatedIdentifier" value="target.value"}} data-test-related-identifier />
+  <span class="input-group-btn">
+    <BsButton @outline={{true}} @onClick={{action "deleteRelatedIdentifier" index}}>{{fa-icon "trash"}}</BsButton>
+  </span>
+</div>
+{{else}}
+<input type="text" class="form-control related-identifier-field" value={{fragment.relatedIdentifier}} oninput={{action "updateRelatedIdentifier" value="target.value"}} data-test-related-identifier />
+<hr />
+
+{{/if}}
+
+<div
+  class="help-block help-block-fragment related-identifier-field no-error {{if (not fragment.relatedIdentifier) 'no-success'}}">
```

Comment:
Can this be made more readable, e.g. by linking to our documentation instead of listing all allowed related identifier types?